ui/projects/spline-shared/graph/src/components/spline-container/spline-container.component.ts

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2021 ABSA Group Limited
+ * Copyright 2023 ABSA Group Limited
```

Comment:
Once added, copyright date must never be updated. Only additional copyright notices are allowed to be added, but we don't that either. We simply leave the original date unchanged.